syntax = "proto3";
package rl;

import "common.proto";

service ExperimentService {

  // Start a new experiment
  rpc StartExperiment(ExperimentConfig) returns (Status);

  // Stop experiment across system
  rpc StopExperiment(ExperimentConfig) returns (Status);

  // Query current status
  rpc GetExperimentStatus(ExperimentConfig) returns (Status);
}
